app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewActivity.kt

```diff
@@ -79,17 +76,7 @@ class PledgedProjectsOverviewActivity : AppCompatActivity() {
                 val showEmptyState = ppoCardPagingSource.loadState.refresh is LoadState.NotLoading && ppoCardPagingSource.itemCount == 0
 
                 KickstarterApp(
-                    useDarkTheme =
-                    if (darkModeEnabled) {
-                        when (theme) {
-                            AppThemes.MATCH_SYSTEM.ordinal -> isSystemInDarkTheme()
-                            AppThemes.DARK.ordinal -> true
-                            AppThemes.LIGHT.ordinal -> false
-                            else -> false
-                        }
-                    } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-                        isSystemInDarkTheme()
-                    } else false
+                    useDarkTheme = isDarkModeEnabled(env = env)
```

Comment:
noticed this was calling this:
`@Composable
fun Context.isDarkModeEnabled(env: Environment): Boolean {
    val darkModeEnabled = env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_DARK_MODE_ENABLED) ?: false
    val theme = env.sharedPreferences()
        ?.getInt(SharedPreferenceKey.APP_THEME, AppThemes.MATCH_SYSTEM.ordinal)
        ?: AppThemes.MATCH_SYSTEM.ordinal

    return if (darkModeEnabled) {
        when (theme) {
            AppThemes.MATCH_SYSTEM.ordinal -> isSystemInDarkTheme()
            AppThemes.DARK.ordinal -> true
            AppThemes.LIGHT.ordinal -> false
            else -> false
        }
    } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
        isSystemInDarkTheme() // Force dark mode uses system theme
    } else false
}`

so i just inlined it